docs/Ivan_CLI.md

```diff
@@ -0,0 +1,240 @@
+use the list subcommand to get an overview of the resources you have running:
+```bash
+$ inductiva resources list
+Active Resources:
+Name                           Machine Type    Elastic    Type        # machines    Disk Size in GB  Spot    Started at (UTC)
+-----------------------------  --------------  ---------  --------  ------------  -----------------  ------  ------------------
+api-abs7c8hwccu0v6cppo9nc40z6  c2-standard-30  False      standard             5                 70  True    01 Feb, 23:06:52
+api-mym74u9i9xppi9ngzze3xueae  c2-standard-8   False           mpi             4                 70  True    02 Feb, 00:29:53
+```
+
+Finally, the CLI also allows to terminate the resources that are no longer required with
+the `terminate` subcommand. Users can either choose a specific resource with the
+`--name` flag or terminate all the resources with the `--all` flag. Any of the steps
+require confirmation from the user before proceeding.
+```bash
+$ inductiva resources terminate --all
+Confirm the termination of all active machines? (y/[N]) y
+Terminating all active computational resources.
+Terminating MachineGroup(name="api-abs7c8hwccu0v6cppo9nc40z6"). This may take a few minutes.
+Machine Group api-abs7c8hwccu0v6cppo9nc40z6 with c2-standard-30 machines successfully terminated in 0:01:07.
+Terminating MachineGroup(name="api-mym74u9i9xppi9ngzze3xueae"). This may take a few minutes.
+Machine Group api-mym74u9i9xppi9ngzze3xueae with c2-standard-8 machines successfully terminated in 0:01:12.
+```
+
+### Task Management
+
+The power of Inductiva API lies on the ability to launch multiple simulations remotely.
+As an easy to track them, the CLI provides a `list` subcommand that provides information
+about the last tasks launched by the user. If you wish, you can also just track
+a specific task via its `--id`. Examples of usage:
+
+```bash
+$ inductiva tasks list -n 4
+ID                         Simulator    Status    Submitted         Started           Computation Time    Resource Type
+-------------------------  -----------  --------  ----------------  ----------------  ------------------  ---------------
+s9evg758ieteqzvg7p6hnuqcw  swash        started   02 Feb, 00:52:10  02 Feb, 00:52:14  *0:00:19            c2-standard-4
+tc7cwuer45kfzuw8t93r6dxa8  swash        success   01 Feb, 23:52:44  01 Feb, 23:52:49  0:25:26             c2-standard-4
+hzgk5ngzk28a39qa7mesv0snk  swash        success   01 Feb, 23:45:19  01 Feb, 23:45:19  0:25:51             c2-standard-4
+mjnb8c7i8bfppgmu2y1zd1o7f  swash        success   01 Feb, 23:24:19  01 Feb, 23:25:08  0:09:43             c2-standard-30
+```
+
+One cool tip is to use a [watch method](https://www.geeksforgeeks.org/watch-command-in-linux-with-examples/) to recurrently monitor the task information.
+For example in Linux or Mac, one can monitor the task information every 10 seconds with:
+```bash
+$ watch -n 10 inductiva tasks -id tc7cwuer45kfzuw8t93r6dxa8
```

Comment:
This should be a flag we provide. We should need to invoke watch (@sergiomsantos : wdyt?)

Reply:
Already discussed this with him, and we will investigate the next steps. Opening a task on it!